openshift/03-buildconfig-template.yml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: zync-buildconfig
+objects:
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    labels:
+      app: zync
+    name: zync
+  spec:
+    output:
+      to:
+        kind: ImageStreamTag
+        name: zync:latest
+    runPolicy: Serial
+    source:
+      git:
+        uri: https://github.com/3scale/zync.git
+      type: Git
+    strategy:
+      sourceStrategy:
+        from:
+          kind: ImageStreamTag
+          name: ruby-24-centos7:latest
+      type: Source
+    triggers:
+    - github:
+        secret: "${GITHUB_SECRET}"
+      type: GitHub
+    - type: ImageChange
+    - type: ConfigChange
```

Comment:
I Think it would be good to have a postCommit like we have in https://github.com/3scale/apicast-cloud-hosted/blob/master/openshift/03-build-config.yml#L10-L15 

Maybe @mikz could point us to a good postCommit